Makefile

```diff
@@ -107,6 +107,10 @@ ifeq ($(ENABLE_KUBE_API),true)
 	SERVICE_REPLICAS_COUNT=1
 	AUTH_TYPE=local
 endif
+ifdef BIP_BUTANE_CONFIG
```

Comment:
Something I just realized, it's missing in the documentation for env-vars
Can you add it in an appropriate place? (I think in https://github.com/openshift/assisted-test-infra/blob/009de89225024572fe99174beded1ff058fb0540/README.md#L127)

Reply:
Good idea, updated